pkg/util/util.go

```diff
@@ -508,3 +508,16 @@ func GetClient(req *http.Request) string {
 	}
 	return "unknown"
 }
+
+func FileExists(filename string) (bool, error) {
+	file, err := os.Open(filename)
```

Comment:
in a different context, it was pointed out to me that you could use ```os.Stat``` here and save yourself having to close the file.